techniques/system/common/1.0/promises.st

```diff
@@ -28,11 +28,10 @@ body common control
         output_prefix => "rudder";
 
         inputs => {
-          @{va.ncf_inputs},
-          &INPUTLIST&
+          @{va.ncf_inputs}
```

Comment:
I'm pretty sure you still need to include rudder_directives.cf and rudder_system_directives.cf here, otherwise they won't be loaded.